package/asset_host.js

```diff
@@ -1,11 +1,10 @@
 const config = require('./config')
 const { resolve } = require('path')
 
-function removeOuterSlashes(string) {
-  return string.replace(/^\/*/, '').replace(/\/*$/, '')
-}
+const removeOuterSlashes = string =>
```

Comment:
Do you prefer this syntax for any particular reason? Personally, I find it much harder to read. And `=>`  != ES6 function. Their behavior is slightly different although it doesn't matter in this case.

To me, this is like changing:
```js
class Environment {
  constructor() {
    ...
  }
}
```
to:
```js
const Environment = class {
  constructor() {
    ...
  }
}
```

Reply:
Nope, just little less curly's `{}` and parans `()` for one liner functions 😄

Reply:
Yeah, for some cases it doesn't do much though.

Reply:
Happy to change it back if you don't like it 😉  although at some places we have fat arrows so would be ideal to make it consistent in that sense